README.md

```diff
@@ -28,3 +28,25 @@ The intention behind creating this new cookie space is the following:
       1. TBD on whether we just just make the endpoint and a config to enter redirect urls into or the whole code base available.
```

Comment:
TBD going to be filled in?

Reply:
TBD should be omitted, we've decided to open source the code itself. the discussion here was whether to hand out the sharedid.org/usync endpoint, or the code itself